roles/maxscale/tests/0005_default.2.5.yml

```diff
@@ -0,0 +1,19 @@
+---
+
+- name: "{{ test }}"
+  hosts: debian:!debian.jessie
+  become: true
+  tasks:
+
+    - block:
+      - import_tasks: pre_tasks/disable_systemd.yml
```

Comment:
We can remove this task if targeted version is not jessie compatible.

Reply:
yeah thanks